### ISSUE TITLE
Questions for the client

### ISSUE THREAD

**Issue by navx2810** [navx2810/gbs-grm#5]
- In the few ways you have to schedule an appointment, which would you say happen the most often?
- How do you keep track of appointments now?
- Who's your favorite customer and why?
- What types of animals do you deal with?
- What services do you offer?

**Comment by navx2810** [comment]
I'll keep this updated with more questions as I think of them. Please feel free to add some as comments if you think of anything else worthwhile to you. Keep in mind, the client isn't technically savvy, try and word your question in a way she can answer and extrapolate your information from that.

**Comment by navx2810** [comment]
## Which scheduling option happens the most?
This question will give us insight as to which method people will use. If people prefer to walk in to schedule an appointment, they will probably not be using the app, but the groomers will. In that case, we have to think which one to focus on.

## How do you track appointments now?
Part of the process behind making a system for someone, is replacing the older system. If the newer system is too foreign or hard to use, veteran employees will stick with what they know and have used in the past. In that instance, you want to design to software to enhance what they already do while you attempt to replace it.

## Favorite customer?
This will give us insight as too what her reoccurring customers are like. If she's an elderly lady, we can't assume she'll be able to use an app, so again, we'd put a heavy focus on designing around the employee interaction.

**Comment by Aphemra** [comment]
## What specific species of pets do you cater to?
When it comes to customers adding pets into the Customer Pet Interface, it would be important to know whether the groomer deals with more than just cats and dogs as 'species' would be an important piece of information for both customer and groomer to know at a glance.

**Comment by navx2810** [comment]
@Aphemra, good point. They might even offer services that don't fit into a platform.
